rotkehlchen/api/v1/resources.py

```diff
@@ -1271,11 +1272,13 @@ def put(
             self,
             validator_index: Optional[int],
             public_key: Optional[Eth2PubKey],
+            ownership_percentage: FVal,
             async_query: bool,
     ) -> Response:
         return self.rest_api.add_eth2_validator(
             validator_index=validator_index,
             public_key=public_key,
+            ownership_proportion=ownership_percentage,
```

Comment:
This naming change from ownership_percentage to ownership_proportion is intended. I reused the filter we have for underlying assets where we pass the proportion using %. So for the reader to know what is being used in each place I named the variable from/to the frontend `ownership_percentage` and the one used in the backend (in the range 0-1) as `ownership_proportion`

Reply:
I don't really see why you would change percentage to proportion. We use the term `weight` for the underlying assets. So I don't see the connection at all. Having different name in backend and frontend is more confusing.